tests/unit/test_pool.py

```diff
@@ -198,6 +213,63 @@ def test_get_non_expired(self):
             self.assertTrue(session._exists_checked)
             self.assertFalse(pool._sessions.full())
 
+    def test_spans_bind_get(self):
```

Comment:
can we add one more negative case for fixedsizepool get method?
scenario - Session pool is empty, and there is a get call on the pool. In this case it will wait until timeout and then fail

Reply:
Done!